### PR TITLE
Fix README build instructions; default to hashless.

### PR DESCRIPTION
**Pull request by cgyarvin**

### CODE REVIEW

**Comment by ohAitch** [comment]
I think you want to target this at `urbit:release-candidate` XD

**Comment by joemfb** [comment]
This PR includes submodule changes that revert those sub-projects to earlier commits.

File: vere/main.c [hunk 1]
```diff
@@ -81,7 +81,7 @@ _main_getopt(c3_i argc, c3_c** argv)
   u3_Host.ops_u.fog = c3n;
   u3_Host.ops_u.gab = c3n;
   u3_Host.ops_u.git = c3n;
-  u3_Host.ops_u.has = c3n;
+  u3_Host.ops_u.has = c3y;
```
Comment (joemfb) [inline vere/main.c]: In #1042, battery hashing is disabled for fake-keyed ships. Is that sufficient, or do you want it to be opt-in everywhere?

**Comment by joemfb** [comment]
closing in favor of #1068

**Pull request closed without merging**